### PR TITLE
[Docs] create cypresstests

### PR DESCRIPTION
**Pull request by sstraatemans**
Adding cypress tests

## Check off the following:

- [x] I have reviewed my changes and run the appropriate tests.
- [x] I have have run `rush change` to add the appropriate change logs.
- [ ] I have added/edited docs.
- [ ] I have added tutorials.
- [ ] I have double checked and DEFINITELY added docs.

### CODE REVIEW

**Review by isaozler — APPROVED** [review]
looks great @sstraatemans

File: packages/apps/kadena-docs/src/components/Layout/components/SideMenu/MenuCard.tsx [hunk 1]
```diff
@@ -8,12 +8,14 @@ export interface IMenuCardProps {
   idx: number;
   ref?: React.ForwardedRef<HTMLDivElement>;
   onClick?: MouseEventHandler<HTMLUListElement>;
+  cy?: string;
 }
 
 export const MenuCard: FC<IMenuCardProps> = React.forwardRef(
-  ({ children, onClick, active, idx = 0 }, ref) => {
+  ({ children, onClick, active, idx = 0, cy }, ref) => {
```
Comment (ash-vd) [inline packages/apps/kadena-docs/src/components/Layout/components/SideMenu/MenuCard.tsx:15]: Maybe make this more explicit, like dataCy? For a second I thought we were passing some kind of cypress instance here

File: packages/apps/kadena-docs/src/utils/hasSameBasePath.ts [hunk 2]
```diff
@@ -2,8 +2,13 @@ export const hasSameBasePath = (
   pathname1?: string,
   pathname2?: string,
 ): boolean => {
+  let basePathLength = 3;
+  if (process.env.NEXT_PUBLIC_APP_DEV === 'test') {
```
Comment (ash-vd) [inline packages/apps/kadena-docs/src/utils/hasSameBasePath.ts:6]: Is this just for very explicit testing, or can we use `NODE_ENV` here?

Comment (sstraatemans) [reply packages/apps/kadena-docs/src/utils/hasSameBasePath.ts:6]: in Next the NODE_ENV is set explicitly by Next itself

**Review by ash-vd — APPROVED** [review]

**Pull request merged**